cirq/google/calibration/workflow.py

```diff
@@ -148,7 +159,10 @@ def make_floquet_request_for_circuit(
             across many circuits.
 
     Returns:
-        Instance of CircuitFloquetPhasedFSimCalibrationRequests.
+        Tuple of:
+          - Circuit together with its calibration metadata connecting to the characterized requests,
```

Comment:
"Circuit and its mapping from moments to indices into the list of calibration requests (the second returned value)"

There's no worry of being too obvious here :)

Reply:
Done.